FIAT/hdiv_trace.py

```diff
@@ -16,14 +16,19 @@
 # along with FIAT. If not, see <http://www.gnu.org/licenses/>.
 
 from __future__ import absolute_import, print_function, division
+from six import iteritems, itervalues
+from six.moves import range
 
 import numpy as np
-from FIAT.discontinuous_lagrange import DiscontinuousLagrange
-from FIAT.reference_element import ufc_simplex
+from FIAT.reference_element import (ufc_simplex, POINT,
+                                    LINE, QUADRILATERAL,
+                                    TRIANGLE, TETRAHEDRON,
+                                    TENSORPRODUCT)
 from FIAT.functional import PointEvaluation
 from FIAT.polynomial_set import mis
-from FIAT import FiniteElement
-from FIAT import dual_set
+from FIAT import (FiniteElement, dual_set,
+                  DiscontinuousLagrange,
+                  TensorProductElement)
```

Comment:
I hope these won't lead to circular import problems. I slightly prefer avoiding import from parent packages.